test/pycheck/temporary_table_test.py

```diff
@@ -0,0 +1,21 @@
+from .utils import Cursor, PG_MAJOR_VERSION
+
+import pytest
+import psycopg.errors
+
+
+@pytest.mark.skipif(
+    "PG_MAJOR_VERSION < 15", reason="SET ACCESS METHOD was introduced in PG15"
+)
+def test_temporary_table_set_access_method(cur: Cursor):
+    cur.sql("CREATE TEMP TABLE t(a int)")
```

Comment:
In the previous test the duckdb access method was the default. In this test it is not. Which is why the first `SET ACCESS METHOD` call is passing.
```suggestion
    cur.sql("CREATE TEMP TABLE t(a int) USING duckdb")
```

Reply:
Ok, that's done